modules/host/net.go

```diff
@@ -22,6 +23,9 @@ func (h *Host) listen() {
 // TODO: maintain compatibility
 func (h *Host) handleConn(conn net.Conn) {
 	defer conn.Close()
+	// Set an initial duration that is generous, but finite. RPCs can extend
+	// this if so desired.
+	conn.SetDeadline(time.Now().Add(5 * time.Minute))
```

Comment:
extending needs to be managed carefully to prevent DoS, but we're always safer with it than without it. The tricky spots are wherever we call `SetDeadline` in a loop, which for now is only `rpcRevise`.

Now that I think of it, it *is* still possible to consume a host connection indefinitely, thanks to this branch: https://github.com/NebulousLabs/Sia/pull/834/files#diff-58607cff14906af27cb7324938c37732R339
I can address this in a later PR.